modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/MeResourceManager.java

```diff
@@ -409,7 +409,6 @@ public SCIMResponse updateWithPATCH(String existingId, String scimObjectString,
         }
     }
 
-
```

Comment:
Let's remove the deleted line.

Reply:
The removed line was an extra new line added from a previous PR. Hope it's okay to remove it with this PR.